lib/ansible/modules/cloud/docker/docker_node_facts.py

```diff
@@ -52,31 +64,39 @@
 '''
 
 EXAMPLES = '''
+- name: Get info on all nodes
+  docker_node_facts:
+  register: result
+
 - name: Get info on node
   docker_node_facts:
     name: mynode
   register: result
 
+- name: Get info on list of nodes
+  docker_node_facts:
+    name:
+      - mynode1
+      - mynode2
+  register: result
+
+- name: Get info on host if it is Swarm Manager
+  docker_node_facts:
+    self: true
+  register: result
 '''
 
 RETURN = '''
-exists:
+nodes_facts:
     description:
-      - Returns whether the node exists in docker swarm cluster.
-    type: bool
+      - Facts representing the current state of the nodes. Matches the C(docker node inspect) output.
+      - Will be C(none) if node does not exist.
+      - Will contain multiple entries if more than one node provided in I(name).
```

Comment:
```suggestion
      - Can contain multiple entries if more than one node provided in I(name), or I(name) is not provided.
```
It could be that only one node exists of the ones specified :) And if `name` is not specified, all nodes will be returned.